server/system/middleware/sessionMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import * as session from 'express-session'
+import connectPgSimple from 'connect-pg-simple'
+
+const db = require('@server/db/db')
+
+const pgSession = connectPgSimple(session)
+
+const sessionOptions = {
+  secret: process.env.FOO_COOKIE_SECRET,
```

Comment:
add this to the README and env.template // use ProcessUtils

Reply:
ok, but I would call the env variable something like SESSION_ID_COOKIE_SECRET (secret to sign the session ID cookie)